components/Player/Player.tsx

```diff
@@ -31,7 +31,7 @@ const Player: React.FC<IPlayerProps> = ({
   children
 }) => {
   const initialTracks = useMemo(
-    () => (Array.isArray(audio) ? audio : [audio]),
+    () => (audio && (Array.isArray(audio) ? audio : [audio])) || [],
```

Comment:
```(audio && (Array.isArray(audio) ? audio : [audio])) || []```
ok, so if `audio` is undefined, then will return empty array
if audio is defined, but not an array, will wrap into an array
if audio is defined and is an array, return that array

lgtm!